src/ILVerification/src/Verifier.cs

```diff
@@ -23,15 +23,19 @@ public class Verifier
             new Lazy<ResourceManager>(() => new ResourceManager("FxResources.ILVerification.SR", typeof(Verifier).GetTypeInfo().Assembly));
 
         private ILVerifyTypeSystemContext _typeSystemContext;
+        private VerifierOptions _verifierOptions;
 
-        public Verifier(IResolver resolver)
+        public Verifier(IResolver resolver, VerifierOptions verifierOptions) : this(new ILVerifyTypeSystemContext(resolver), verifierOptions) { }
+
+        internal Verifier(ILVerifyTypeSystemContext context, VerifierOptions verifierOptions)
         {
-            _typeSystemContext = new ILVerifyTypeSystemContext(resolver);
+            _typeSystemContext = context;
+            _verifierOptions = verifierOptions ?? GetDefaultVerifierOptions();
         }
 
-        internal Verifier(ILVerifyTypeSystemContext context)
+        private VerifierOptions GetDefaultVerifierOptions()
         {
-            _typeSystemContext = context;
+            return new VerifierOptions { IncludeMetadataTokensInErrorMessages = false };
```

Comment:
`new VerifierOptions()` should create object with good defaults. No need to set the properties.

Reply:
Ok!I thought it was more clear.

Reply:
done thank's Jan!